resources/saas-boost-web.yaml

```diff
@@ -129,10 +126,8 @@ Resources:
             Value: !Ref ApiGatewayUrl
           - Name: REACT_APP_CLIENT_ID
             Value: !Ref AdminWebClientId
-          - Name: REACT_APP_COGNITO_USERPOOL
-            Value: !Ref UserPoolId
-          - Name: REACT_APP_COGNITO_USERPOOL_BASE_URI
-            Value: !Ref UserPoolBaseUri
+          - Name: REACT_APP_OIDC_HOST
+            Value: !Ref OidcHost
```

Comment:
this isn't necessary, only the issuer is actually needed. but I can change this when I build on top of this commit.